// azure-devops-extension-api/AdvancedSecurity.Reporting/Reporting.js
/*
 * ---------------------------------------------------------
 * Copyright(C) Microsoft Corporation. All rights reserved.
 * ---------------------------------------------------------
 */
define(["require", "exports"], function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    var AlertType;
    (function (AlertType) {
        /**
         * The code has an unspecified vulnerability type
         */
        AlertType[AlertType["Unknown"] = 0] = "Unknown";
        /**
         * The code uses a dependency with a known vulnerability.
         */
        AlertType[AlertType["Dependency"] = 1] = "Dependency";
        /**
         * The code contains a secret that has now been compromised and must be revoked.
         */
        AlertType[AlertType["Secret"] = 2] = "Secret";
        /**
         * The code contains a weakness determined by static analysis.
         */
        AlertType[AlertType["Code"] = 3] = "Code";
        /**
         * The code uses a dependency with potential license incompliance.
         */
        AlertType[AlertType["License"] = 4] = "License";
    })(AlertType = exports.AlertType || (exports.AlertType = {}));
});
